conmon/conmon.c

```diff
@@ -152,11 +175,11 @@ int main(int argc, char *argv[])
 	/* Create the container */
 	if (terminal) {
 		snprintf(cmd, CMD_SIZE,
-			 "%s create %s --pid-file pidfile --console %s",
-			 runtime_path, cid, slname);
+			 "%s create %s %s --pid-file %s --console %s",
```

Comment:
> The way I would read create <arg1> <arg2> (which is admittedly quite confusing and should be changed IMO because the meaning is hidden behind making it look like a CLI UX)

The `UX` part of this CLI UX PR bothers me a little: Just as a reminder, the OCI runtime spec CLI is a major integration point between any OCI runtime and Docker (containerd-shim). It goes way beyond a simple user experience as its definition will make anyone whose business relies on an container runtime being OCI compatible work or not with Docker.

 > The way I would read create <arg1> <arg2> (which is admittedly quite confusing and should be changed IMO because the meaning is hidden behind making it look like a CLI UX) is that it means that there is an operation called "create" which has two mandatory arguments

That's one interpretation, and I agree it's quite confusing. Again, for any of us OCI runtime implementers, we can only interpret it as the runtime CLI specification because:
1. There are no other CLI specification at the moment
2. Docker calls into us through an actual CLI

I'm glad there's a pending runtime CLI specification PR, this should clarify things a lot.

> there is an operation called "create" which has two mandatory arguments

Regardless of the way you interpret it, runC is currently not compatible with it as the `--bundle` switch is optional.